src/Umbraco.Web.UI.Client/src/views/media/restore.html

```diff
@@ -0,0 +1,26 @@
+<div ng-controller="Umbraco.Editors.Media.RestoreController">
+	<div class="umb-dialog-body">
+		<umb-pane>
+
+			<p class="abstract" ng-hide="error != null || success == true">
+				Restore <strong>{{currentNode.name}}</strong> to under <strong>{{target.name}}</strong>?
```

Comment:
The texts in this view should be retreived from the language files using the localize directive